### PR TITLE
Adding a config file

### PR DESCRIPTION
**Pull request by pierre-alain-b**
All the changes would help another user deploy the app with all parameters now set up in a config file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,48 +1,51 @@
 # About
-[Uguu.se](http://uguu.se) source code, stores files and deletes after X amount of time.
+Temp file sharing application source code, stores files and deletes after X amount of time. Forked from Uguu.se available [here](https://github.com/nokonoko/uguu).
```
Comment (pierre-alain-b) [inline README.md]: Maybe this change should not be in the PR, sorry...

**Comment by alucard0134** [comment]
holy commits batman, and yea, there are some commits in this PR (ex change uguu.se to uguu) that should be in a different PR for discussion imo

**Comment by nokonoko** [comment]
It looks good and I'll merge it, thank you.

**Comment by nokonoko** [comment]
Also seems to work fine live, great, maybe I should of tested it first but ehhh haha no.

**Comment by nokonoko** [comment]
Works fine live after some tweaking so a sub domain would work to serve files, added this to the configuration however.

**Comment by pierre-alain-b** [comment]
Thank you for all the changes. I will mark my fork as not relevant anymore and forward to your version!

**Comment by alucard0134** [comment]
side note: git rebase is a neat thing to merge and change commit messages

**Comment by pierre-alain-b** [comment]
Definitely I should have a look at that. Must admit I've never used git rebase. Will cure this soon.
Thanks for pointing it out!

**Comment by alucard0134** [comment]
trust me we all wish @nokonoko used it

**Pull request merged**